ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -84,10 +91,29 @@ export class ImportCommand extends IronfishCommand {
 
     process.stdin.off('data', onData)
 
-    return JSONUtils.parse<AccountImport>(data)
+    try {
+      const decodedWords = bech32m.bech32m.decode(data).words
+      const decodedBytes = bech32m.bech32m.fromWords(decodedWords)
+      return JSONUtils.parse<AccountImport>(decodedBytes.toString())
+    } catch (e) {
+      return JSONUtils.parse<AccountImport>(data)
+    }
   }
 
   async importTTY(): Promise<AccountImport> {
+    const bech32 = await CliUx.ux.prompt(
```

Comment:
Can use more interactive selector rather than requiring user to read entire message (with examples in text output)
https://github.com/iron-fish/ironfish/blob/027b96c406bb15e754e3829aef87c373313d382f/ironfish-cli/src/utils/asset.ts#L49-L56

Reply:
Nice, I like that much better.